packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -67,27 +89,30 @@ const COUNT_COL = new GridColumn({
 // the column index for cell values and cell messages does not include either the selection
 // column or the row number column, so we adjust the value passed to <Cell> to accommodate.
 function inputCellFactory(
-    model: QueryGridModel,
+    queryInfo: QueryInfo,
     editorModel: EditorModel,
-    allowSelection?: boolean,
-    hideCountCol?: boolean,
-    columnMetadata?: EditableColumnMetadata,
-    readonlyRows?: List<any>,
-    onCellModify?: () => any
+    allowSelection: boolean,
+    hideCountCol: boolean,
+    columnMetadata: EditableColumnMetadata,
+    readonlyRows: List<any>,
+    selectCell: (colIdx: number, rowIdx: number, selection?: SELECTION_TYPES, resetValue?: boolean) => void,
+    modifyCell: (colIdx: number, rowIdx: number, newValues: ValueDescriptor[], mod: MODIFICATION_TYPES) => void,
+    focusCell: (colIdx: number, rowIdx: number, clearValue?: boolean) => void,
+    clearSelection: () => void,
+    inDrag_: () => boolean
```

Comment:
I saw some of these similar property definitions in a couple places. Any utility in passing around a single `actions` prop instead?